collect_app/src/main/java/org/odk/collect/android/activities/FileManagerTabs.java

```diff
@@ -65,19 +66,21 @@ protected void onCreate(Bundle savedInstanceState) {
         int fontsize = Collect.getQuestionFontsize();
 
         ViewGroup rllf = (ViewGroup) tw.getChildAt(0);
-        mTVFF = getTextViewChild(rllf);
+        TextView mTVFF = getTextViewChild(rllf);
```

Comment:
I like your change to local variables very much BUT it means the name should also change because the `m` prefix convention is used to indicate members/fields.

It would be best to issue separate pull requests for fixes and refactoring. That makes the review easier and reduces the risk of making errors. For this one, go ahead and just make the change here. You'll probably want to change the variable name to something a bit more descriptive in addition to removing the `m` prefix.

Then, if you want to take on a bigger refactor of this class, please do! Even just a pass improving names would be great.